security-oauth2/src/main/java/io/micronaut/security/oauth2/endpoint/authorization/response/AuthorizationErrorResponseExceptionHandler.java

```diff
@@ -30,8 +34,37 @@
 @Singleton
 public class AuthorizationErrorResponseExceptionHandler implements ExceptionHandler<AuthorizationErrorResponseException, MutableHttpResponse<?>> {
 
+    private final ErrorResponseProcessor<?> responseProcessor;
+
+    /**
+     * Default constructor.
+     *
+     * @deprecated See @{link #AuthorizationErrorResponseExceptionHandler(ErrorResponseProcessor)}.
+     */
+    @Deprecated
```

Comment:
remove deprecated code. This is going to be merged now into the next mayor branch.